packages/extension-color/src/index.ts

```diff
@@ -1,5 +1,6 @@
-import { Color } from './color.js'
+// @ts-ignore
+import { Color, ColorOptions } from '@tiptap/extension-text-style/color'
```

Comment:
Notice that we only load from the sub dir export, meaning the minimum amount of code will be loaded (it will be pretty much the same as what the package was exporting before)